src/ORM/Query.php

```diff
@@ -30,7 +31,7 @@
  * required.
  *
  */
-class Query extends DatabaseQuery implements JsonSerializable
+class Query extends DatabaseQuery implements JsonSerializable, QueryInterface
```

Comment:
You can put docs above/below the parent docs. We do that in a few classes already I think.